### DISCUSSION TITLE
Controlling fans on Corsair Fan Controller

### DISCUSSION THREAD

**Discussion by FTSCOOP** [Rem0o/FanControl.Releases#838] [Q&A]
CORSAIR Crystal Series 680X RGB has a corsair controller.  Has anybody had any luck being able to control them.

**Comment by Bucketsmith** [comment]
https://github.com/iJacks1980/FanControl.CommanderPRO
Use this!

**Reply by guymanley** [reply]
Thanks. I just got the commander pro and had a mini panic attack when it didn't see those fans in FanControl.
